packages/appstore/src/constants/trading-hub-content.tsx

```diff
@@ -13,39 +14,67 @@ type TTradingHubContents = Record<'step1' | 'step2' | 'step3' | 'step4' | 'step5
 
 export const trading_hub_contents: TTradingHubContents = {
     step1: {
-        component: 'Step1',
+        component: <StaticDashboard />,
         footer_header: localize('Welcome to Trading Hub'),
         footer_text: localize('This is your new dashboard'),
         has_next_content: false,
     },
     step2: {
-        component: 'Step2',
+        component: <StaticDashboard is_cfd_text_blurry is_options_text_blurry is_platformlauncher_blurry />,
         footer_header: localize('Trading accounts'),
         footer_text: localize(
             'These are the trading accounts available to you. You can click on an account to find out more.'
         ),
         has_next_content: false,
     },
     step3: {
-        component: 'Step3',
+        component: <StaticDashboard is_cfd_item_blurry is_options_item_blurry is_platformlauncher_blurry />,
         footer_header: localize('CFDs, Options or Multipliers'),
         footer_text: localize('You can choose between CFD trading accounts or Options and Multipliers accounts'),
         has_next_content: false,
     },
     step4: {
-        component: 'Step4',
+        component: (
+            <StaticDashboard is_cfd_text_blurry is_options_text_blurry is_platformlauncher_blurry is_button_animated />
+        ),
         footer_header: localize('‘Get’ the accounts you want'),
         footer_text: localize('Click the ‘Get’ button to create an account'),
         has_next_content: false,
     },
     step5: {
-        component: 'Step5',
```

Comment:
1. Top-up and Trade buttons color should change 
2. `More Derived accounts` and `More Financial accounts` should be added
3. in the Options part the grey background should be same for `US Dollar`  and `platform lunchers` and there shouln't be white space between them
4. other Options accounts rather than `US Dollar` should be `lowlighted`

Test link:
![image](https://user-images.githubusercontent.com/82078941/192283634-072702f4-8473-4d7d-815f-d85284106e02.png)

Design: 
![image](https://user-images.githubusercontent.com/82078941/192283707-a5f744bc-5fbd-4935-93d4-203643ce5995.png)